src/partials/nav.hbs

```diff
@@ -11,7 +11,7 @@
         </nav>
       </div>
       <div class="nav-collapse">
-        <button id="nav-collapse-toggle"><span></span></button>        
+        <button id="nav-collapse-toggle"><span></span></button>
```

Comment:
Empty span in button is not necessary i think

Reply:
![image](https://github.com/PSI-Polska/antora-ui-psi/assets/130150009/f0a3415b-8da3-4b2d-a081-0a3357ca4dcf)
This span is needed - look at nav.css -> .nav-collapse button span {